docs/glossary.md

```diff
@@ -0,0 +1,19 @@
+| napari term        | description                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                  | synonym/closely related term                                                |
```

Comment:
For JupyterBook to recognize your document, it must have a title.

```suggestion
# Glossary

| napari term        | description                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                  | synonym/closely related term                                                |
```

In addition, it needs to be added to the table of contents `_toc.yml` if you want to preview it locally, or if you want to see it in the historical docs pages hosted at napari.org/docs - I don't think that's critical here as the actual table of contents rendered in the final napari.github.io site is defined in the napari.github.io repo, not here. Once this document is ready we can add it there.